include/sound/sof.h

```diff
@@ -27,6 +27,9 @@ struct snd_sof_pdata {
 
 	struct device *dev;
 
+	/* indicate how many first bytes shouldn't be loaded into DSP memory. */
```

Comment:
commit message: please clarify what you meant by "put extra information to host"

Provide extra information to host before downloading firmware?

Reply:
changed